core/src/main/java/org/mule/runtime/core/api/exception/FlowExceptionHandler.java

```diff
@@ -35,6 +35,7 @@
   CoreEvent handleException(Exception exception, CoreEvent event);
 
   @Override
+  @Deprecated
```

Comment:
add a note, shouldn't `handleException` also be deprecated?

Reply:
Yes, see the `@Deprecated` annotation above